packages/contracts/development/contracts/IRealityETH_ERC20.sol

```diff
@@ -1,67 +1,160 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
-pragma solidity 0.8.10;
+pragma solidity 0.8.20;
 
-import './IBalanceHolder_ERC20.sol';
+import {IBalanceHolder_ERC20} from "./IBalanceHolder_ERC20.sol";
+import {IERC20} from "./IERC20.sol";
+import {IRealityETHErrors_ERC20} from "./IRealityETHErrors_ERC20.sol";
 
-interface IRealityETH_ERC20 is IBalanceHolder_ERC20 {
-     event LogAnswerReveal (bytes32 indexed question_id, address indexed user, bytes32 indexed answer_hash, bytes32 answer, uint256 nonce, uint256 bond);
-     event LogCancelArbitration (bytes32 indexed question_id);
-     event LogClaim (bytes32 indexed question_id, address indexed user, uint256 amount);
-     event LogFinalize (bytes32 indexed question_id, bytes32 indexed answer);
-     event LogFundAnswerBounty (bytes32 indexed question_id, uint256 bounty_added, uint256 bounty, address indexed user);
-     event LogMinimumBond (bytes32 indexed question_id, uint256 min_bond);
-     event LogNewAnswer (bytes32 answer, bytes32 indexed question_id, bytes32 history_hash, address indexed user, uint256 bond, uint256 ts, bool is_commitment);
-     event LogNewQuestion (bytes32 indexed question_id, address indexed user, uint256 template_id, string question, bytes32 indexed content_hash, address arbitrator, uint32 timeout, uint32 opening_ts, uint256 nonce, uint256 created);
-     event LogNewTemplate (uint256 indexed template_id, address indexed user, string question_text);
-     event LogNotifyOfArbitrationRequest (bytes32 indexed question_id, address indexed user);
-     event LogReopenQuestion (bytes32 indexed question_id, bytes32 indexed reopened_question_id);
-     event LogSetQuestionFee (address arbitrator, uint256 amount);
+/* solhint-disable func-name-mixedcase */
```

Comment:
Public variables with underscores are in the existing ABI and changing it will create otherwise pointless work for anyone else with an integration so I'm inclined to keep them as is, but disable them in the most local scope possible so new stuff is standard unless we decide to change it on purpose.